pkg/kademlia/kademlia.go

```diff
@@ -423,3 +426,9 @@ func randomIDInRange(start, end bucketID) (storj.NodeID, error) {
 	}
 	return randID, nil
 }
+
+// timeTrack tracks how long a function ran for
+func (k *Kademlia) timeTrack(start time.Time, name string) {
+	elapsed := time.Since(start)
+	k.log.Debug("", zap.Stringer(k.routingTable.self.Type.String(), k.routingTable.self.Id), zap.Duration(name, elapsed))
```

Comment:
There's no need to log self.Type and self.Id... this is obvious from the log that you are looking at. It only can be one thing.